RFC-0004-support-distributed-procedure.md

```diff
@@ -0,0 +1,397 @@
+
+## **Expand Procedure Architecture to Support Distributed Execution**
+
+## Proposers
+
+* Dong Wang
+
+## Related Issues
+
+* https://github.com/prestodb/presto/issues/20425
+
+## Summary
+
+Propose design to expand the current procedure architecture in presto, support defining, registering and calling procedures which need to be executed in a distributed way.
```

Comment:
Can you also define what you mean by procedure?

Reply:
Thanks for the suggestion. Strictly speaking, maybe they should be called connector specific system procedures. I'll add some explanations here.